dist/profile/templates/confluence/vhost.conf

```
@@ -2752,6 +2752,8 @@ RewriteRule "^/display/JENKINS/Pipeline\+Plugin$" "https://plugins.jenkins.io/wo
 RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
 RewriteRule "^/display/JENKINS/Workflow\+Plugin$" "https://plugins.jenkins.io/workflow-aggregator" [NC,L,QSA,R=301]
 RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/Gerrit\+Trigger/$" "https://plugins.jenkins.io/gerrit-trigger" [NC,L,QSA,R=301]
```

I think that the trailing slash will not help with the goal to redirect that page.
```suggestion
RewriteRule "^/display/JENKINS/Gerrit\+Trigger$" "https://plugins.jenkins.io/gerrit-trigger" [NC,L,QSA,R=301]
```